src/Styles/icon.scss

```diff
@@ -29,29 +29,32 @@
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
   &.FolderOpen:before {
-    content: "\e956";
+    content: '\e956';
   }
-  &.Bill:before {
-    content: '\e954';
+  &.Timer:before {
```

Comment:
Seems like this was just cut and pasted from the bottom?

Reply:
Yeah some of them got moved, since I refreshed teh entire file from the list generated by icomoon

Reply:
Just trying to avoid unnecessary diff, if that is the case. This makes it easier to avoid bugs, specially the ones where tests can't catch them. (like this case)

Reply:
We should let this one slide, then ensure we follow icomoon's order moving forward then. I must have pasted these on top in the past.

Reply:
It looks like these used to be alphabetized. We should maintain that ordering if there's no other ordering present

Reply:
I agree with Sahas here. The changes in ordering give the impression that the diff is bigger than it is, and hides the actual change

Reply:
I would be okay with letting it slide if we hadn't had bad previous experiences with shipping changes that didn't feel like a big deal, but then they clog up the pipeline for days. 

Probably just paranoia but the consequences in the past have been very real 😄